tests/expectations/tests/call-conv-field.rs

```diff
@@ -37,6 +37,6 @@ impl Default for JNINativeInterface_ {
     fn default() -> Self { unsafe { ::std::mem::zeroed() } }
 }
 extern "stdcall" {
-    #[link_name = "_bar@0"]
+    #[link_name = "bar@0"]
```

Comment:
Yeah, this is due to us not testing on OSX (where we remove an extra underscore from symbols that clang adds). Of course, this is cross-compiled so a `cfg` check is not enough. Let's just remove this change from the PR for now.